addons/account/models/account_invoice.py

```diff
@@ -1605,6 +1607,17 @@ def _default_account(self):
     company_currency_id = fields.Many2one('res.currency', related='invoice_id.company_currency_id', readonly=True, related_sudo=False)
     is_rounding_line = fields.Boolean(string='Rounding Line', help='Is a rounding line in case of cash rounding.')
 
+    line_type = fields.Selection(
```

Comment:
guideline format?

Reply:
``` python
line_type = fields.Selection([
    ('accountable', 'Accountable'),
    ('section', 'Section'),
    ('note', 'Note')], default='accountable')
```
Please add a help attribute on the field. As the field is named ``line_type``, no need to be too specific on the keys.

Reply:
I was specific on the keys for easier greping/replacing them, as the names are also still up to debate.